docs/design/architecture.md

```diff
@@ -3,89 +3,103 @@ title: Architecture
 sidebar_position: 1
 ---
 
-WIP on Notion. 
+Quickwit has been designed from the ground up for cost-efficiency, scalability, and cloud-friendly. 
 
-Quickwit is built on three pillars:
-- the index: a set of data structures tailored to be queried, even on high latency storage;
-- the metastore: stores index metadata and makes them available to all search nodes;
-- the search cluster: provides high availability search, workload distribution, and efficient caching.
+Quickwit core relies on two processes:
 
-[//]: # (Add space with '---' and align image for docusaurus)
+- The Searchers for executing search queries from the REST API.
+- And the Indexers that index data from data sources.
 
----
-<div style={{textAlign: 'center'}}>
+Moreover, Quickwit leverages existing infrastructure by relying on battled-tested technologies for index storage, metadata storage, and ingestion:
 
-![Quickwit Architecture](../assets/images/quickwit-architecture.svg)
+### Index storage
+
+Quickwit stores indexed data on Amazon S3. In a single-server deployment, this can also be your local disk. We plan to support HDFS or any other cloud storage.
```

Comment:
```suggestion
Quickwit stores its indexes's file on Amazon S3. In a single-server deployment, this can also be your local disk. We plan to support HDFS or any other cloud storage.
```